src/utils/ProviderUtils.ts

```diff
@@ -528,6 +562,12 @@ export async function getProvider(chainId: number, logger?: winston.Logger, useC
 
   const nodeMaxConcurrency = Number(process.env[`NODE_MAX_CONCURRENCY_${chainId}`] || NODE_MAX_CONCURRENCY || "25");
 
+  const disableNoTtl = process.env["DISABLE_NO_TTL"] === "true";
+
+  const noTtlBlockDistance = process.env[`NO_TTL_BLOCK_DISTANCE_${chainId}`]
+    ? Number(process.env[`NO_TTL_BLOCK_DISTANCE_${chainId}`])
+    : DEFAULT_NO_TTL_DISTANCE[chainId];
```

Comment:
Minor style nit - IMO preferable to reduce the number of times we have to specify an env var as a string.

```suggestion
  const noTtlBlockDistance = Number(process.env[`NO_TTL_BLOCK_DISTANCE_${chainId}`] ?? DEFAULT_NO_TTL_DISTANCE[chainId]);
```

Reply:
Actually, I think I want this to be undefined if _neither_ the env or the default is defined. So I don't think we want to wrap both in Number() -- does that make sense? I should add a comment/type annotation about this.

Reply:
I'm going to go ahead and merge today, but I'll send you a message IRL so you get a chance to come back to this comment.